midp/background.js

```diff
@@ -91,6 +91,7 @@ Native["com/nokia/mid/s40/bg/BGUtils.maybeWaitUserInteraction.(Ljava/lang/String
 
   // If the page is visible, just start the FG MIDlet
   if (!document.hidden) {
+    profile === 3 && startTimeline();
```

Comment:
Actually, I meant a longer timeout (maybe configurable?) to give the BG MIDlet time to settle.